runtime/plugin/tutor.vim

```diff
@@ -0,0 +1,6 @@
+if exists('g:loaded_tutor_mode_plugin') || &compatible
```

Comment:
I think keeping the differences minimal would be ideal. I plan to write a script to automate the process, though.